keyboards/dz60/keymaps/chrisae9/build_flash.sh

```diff
@@ -0,0 +1,9 @@
+# dfu-programmer atmega32u4 erase --force
+# dfu-programmer atmega32u4 flash /path/to/firmware.hex
+# dfu-programmer atmega32u4 reset
+
+# run this in the qmk_firmware directory
+make dz60:chrisae9
```

Comment:
As a FYI you can now run `make dz60:chrisae9:flash` for a all in one solution.

Reply:
I'll second this.  This will actually call all the commands that you need to compile and flash the keyboard, without the additional stuff.  And more importantly, it will error out and stop flashing, if the compiling fails.   Whereas, this script will flash the old firmware, if it fails.